ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -315,10 +312,11 @@ def _get_columns(self, schema: str, table: str, inspector: Inspector) -> List[Co
         return table_columns
 
     def _get_table_description(self, schema: str, table: str, inspector: Inspector) -> str:
+        description = None
         try:
             table_info: dict = inspector.get_table_comment(table, schema)
-        except NotImplementedError:
-            description: Optional[str] = None
+        except Exception as err:
```

Comment:
can you add details on why this is needed. Which connector failing this

Reply:
Bigquery was failing it, we have resolved it.